Formula/postgresql@95.rb

```diff
@@ -0,0 +1,147 @@
+class PostgresqlAT95 < Formula
+  desc "Object-relational database system"
+  homepage "https://www.postgresql.org/"
+  url "https://ftp.postgresql.org/pub/source/v9.5.5/postgresql-9.5.5.tar.bz2"
```

Comment:
I've just bumped the version of `postgresql@95` to 9.5.6 and added it to this pull request.